src/libs/ajax/data-table-providers/EntityServiceDataTableProvider.ts

```diff
@@ -41,4 +69,20 @@ export class EntityServiceDataTableProvider implements DataTableProvider {
   downloadTsv = (signal: AbortSignal, entityType: string): Promise<Blob> => {
     return Ajax(signal).Workspaces.workspace(this.namespace, this.name).getEntitiesTsv(entityType)
   }
+
+  uploadTsv: uploadTsvFn = async (uploadParams: UploadParameters) => {
```

Comment:
This doesn't need to be "double typed" with both `uploadTsvFn` and parameter types. For readability, I'd suggest 
```ts
uploadTsv = async (uploadParams: UploadParameters): Promise<Response> => {
```